### ISSUE TITLE
An unhandled exception occurred: The target entry-point "ngx-bootstrap-icons" has missing dependencies:  - lodash.camelcase

### ISSUE THREAD

**Issue by Rob4226** [avmaisak/ngx-bootstrap-icons#381]
Not sure why but when trying to use this package I got this error when trying to `ng serve` an Angular project:
```sh
An unhandled exception occurred: The target entry-point "ngx-bootstrap-icons" has missing dependencies:
 - lodash.camelcase
 ```
I deleted package.json, node_modules, and my cache, then tried installing everything again but got same error. I ended up just installing the package it asked for then it worked.
```sh
npm i lodash.camelcase
```

**Comment by avmaisak** [comment]
Hello @Rob4226 . This issue has been fixed in 1.4.6. Please update package! If this issue has not been resolved, please reopen in, or create new.

**Comment by Rob4226** [comment]
@avmaisak Ok cool, looks good now. Thanks!
